libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -266,30 +273,35 @@ void AP_MotorsMatrix::output_armed_stabilizing()
 
     // calculate amount of yaw we can fit into the throttle range
     // this is always equal to or less than the requested yaw from the pilot or rate controller
-    float rp_low = 1.0f;    // lowest thrust value
     float rp_high = -1.0f;  // highest thrust value
```

Comment:
Correct me if I am wrong but I think when I changed the calculation of yaw_alowed on the old line 317 and 319 I removed the need to calculate both rp_low AND rp_high.......
Missed that one didn't I!

Reply:
Yep, `rp_high` also removed.